natgeo/mortar
# [SYTO-1267] Fixed an issue on CTA button when type is link

To test:

- Go to Components
- Search for the link "share this article"
- Verify it has black font color
